.github/workflows/ci.yml

```diff
@@ -1,124 +1,159 @@
 name: CI
 
+env:
+    APPLICATION_ENV: 'development'
+    APPLICATION_STORE: 'DE'
+    PROJECT: 'Payone'
+    DATABASE_VERSION: 10.2
+    DATABASE_HOST: 127.0.0.1
+    DATABASE_PORT: 3306
+    DATABASE_NAME: eu-docker
+    DATABASE_USERNAME: root
+    DATABASE_PASSWORD: secret
+    DATABASE_ROOT_PASSWORD: secret
+    DATABASE_ALLOW_EMPTY_PASSWORD: false
+    DATABASE_CHARACTER_SET: utf8
+    DATABASE_COLLATE: utf8_general_ci
+    MYSQL_ALLOW_EMPTY_PASSWORD: false
+    MYSQL_ROOT_PASSWORD: secret
+    MYSQL_DATABASE: eu-docker
+
 on:
     pull_request:
     push:
         branches:
             - master
     workflow_dispatch:
 
-env:
-    SPRYKER_TESTING_ENABLED: 1
-    APPLICATION_STORE: DE
-    POSTGRES_PORT: 5433
-    DB_USER: postgres
-    DB_NAME: DE_test_zed
-    DB_PASSWORD: pwd
-    MODULE_DIR: module
-    SHOP_DIR: current
-    MODULE_NAME: payone
-
 jobs:
-    test_pgsql:
-        name: "Test / PostgreSQL"
+    setup:
+        name: Setup MariaDB
         runs-on: ubuntu-18.04
-        strategy:
-            fail-fast: false
-            matrix:
-                php: [ '7.3', ]
-                product: [ 'suite' ]
-                composer_preference: ['', '--prefer-lowest']
+
+        steps:
+            - uses: actions/checkout@v2
+            - uses: getong/mariadb-action@v1.1
```

Comment:
Please check mariadb implementation. Once it is defined with getong/mariadb-action@v1.1 and below in services again

Reply:
I will have 

In ConnectionFactory.php line 42:
                             
  Unable to open connection  
                             

In PdoAdapter.php line 82:
                                 
  Unable to open PDO connection  
                                 

In PdoConnection.php line 82:
                                             
  SQLSTATE[HY000] [2002] Connection refused  
  
  if I remove it from services